docs/articles/guides/console-args.md

```diff
@@ -243,6 +243,7 @@ dotnet run -c Release -- --filter * --runtimes net6.0 net8.0 --statisticalTest 5
 * `--platform` the Platform that should be used. If not specified, the host process platform is used (default). AnyCpu/X86/X64/Arm/Arm64/LoongArch64.
 * `--runOncePerIteration` run the benchmark exactly once per iteration.
 * `--buildTimeout` build timeout in seconds.
+* `--preventSleep` prevents the system from entering sleep or turning off the display. None/RequireSystem/RequireDisplay.
```

Comment:
I find it a little strange that the code config uses `WakeLock` while the console arg uses `--preventSleep`. Can the console arg also be `--wakelock` or `--wakeLock`?